Open Judge System/Workers/OJS.Workers.Executors/StandardProcessExecutor.cs

```diff
@@ -211,6 +211,13 @@ public StandardProcessExecutor(int baseTimeUsed, int baseMemoryUsed)
             {
                 result.Type = ProcessExecutionResultType.RunTimeError;
             }
+            else
+            {
+                if (result.ExitCode != 0 && result.ExitCode != -1)
```

Comment:
just add this check litteraly in the above `if(..)` with `||` no need of `else` statement